examples/examples-log4j2/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Is this file used? Most example sub-directories don't contain gradlew.bat and gradlew (Linux shell script). Seems to be redundant?